### PR TITLE
Refresh requirements

### PR DESCRIPTION
**Pull request by jakirkham**
Updates conda-build's requirements based on [it's included `meta.yaml`]( https://github.com/conda/conda-build/blob/3.18.6/conda.recipe/meta.yaml ).

Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jakirkham** [comment]
> jakirkham requested a review from stuertz as a code owner now

Weird. Definitely did **not** do that. Seems GitHub has a mind of its own.

<hr>

Though I did request a review from you Mike. 🙂

**Comment by isuruf** [comment]
@jakirkham, can you add ripgrep too? https://github.com/conda-forge/conda-build-feedstock/issues/102

**Comment by jakirkham** [comment]
Let’s handle that separately. Right now I just want to get the requirements correct, which they are not currently.

**Comment by jakirkham** [comment]
@msarahan, could you give this a quick look please?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,11 +44,12 @@ requirements:
     - py-lief            # [not (win and py2k)]
     - pyyaml
     - scandir            # [py27]
+    - setuptools
     - six
     - glob2 >=0.6
     - pytz
     - tqdm
-    - conda-package-handling >=1.1
+    - conda-package-handling >=1.2
```
Comment (msarahan) [inline recipe/meta.yaml]:
```suggestion
    - conda-package-handling >=1.3
```

**Comment by msarahan** [comment]
bump CPH, otherwise fine

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Should we update [this line]( https://github.com/conda/conda-build/blob/14e89bee1ef6c0506bdee7d2aaae2fc48d231939/conda.recipe/meta.yaml#L48 ) as well?

**Comment by jakirkham** [comment]
Thanks for giving it a look. Can do. 🙂

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Added PR ( https://github.com/conda/conda-build/pull/3587 ).

**Comment by scopatz** [comment]
Looks likt it is failing on mac

**Comment by jakirkham** [comment]
Sounds like some sort of other issue outside of this PR then. Presumably we were already using the latest version of `conda-package-handling`, which is already 1.3+.

**Comment by scopatz** [comment]
Yeah, I am seeing this in staged-recipes too: https://github.com/conda-forge/staged-recipes/pull/8651

**Comment by jakirkham** [comment]
Yeah it affected the JupyterLab 1.0.1 PR too ( https://github.com/conda-forge/jupyterlab-feedstock/pull/158 ).

**Comment by mariusvniekerk** [comment]
Restarted the failed builds and seems to be okay now

**Comment by jakirkham** [comment]
FWIW we probably don't want to merge this until we are ready to go back to the new version of conda-build.

**Comment by mariusvniekerk** [comment]
Or until we have some patches (eeew) that make it work

**Comment by mariusvniekerk** [comment]
I'm not entirely sure what the build errors are on the emulated platforms.  It suggests that there may be some issues with how qemu operates since we moved to 4.0.0-5  Should we roll that back to the last good 3.x release?

**Comment by jjhelmus** [comment]
> I'm not entirely sure what the build errors are on the emulated platforms. It suggests that there may be some issues with how qemu operates since we moved to 4.0.0-5 Should we roll that back to the last good 3.x release?

These errors have been seen in a few other recipes:
* conda-forge/openssl-feedstock#46
* conda-forge/fribidi-feedstock#2

If reverting to qemu 3.x fixes this I would be in favor.

**Comment by mbargull** [comment]
Looks like @mariusvniekerk is spot-on:
```sh
>>> docker run --rm -it --entrypoint='' mbargull/condaforge-linux-anvil-aarch64:qemu-3.1.0-3 /opt/conda/bin/python -c 'import os;os.listxattr(".");print("all good")'
all good
>>> docker run --rm -it --entrypoint='' mbargull/condaforge-linux-anvil-aarch64:qemu-4.0.0-5 /opt/conda/bin/python -c 'import os;os.listxattr(".");print("all good")'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
OSError: [Errno 38] Function not implemented: '.'
```

**Comment by mariusvniekerk** [comment]
Reverted to qemu 3.1 and its now fine

**Comment by jakirkham** [comment]
Ok, feel free to merge then 🙂

**Pull request merged**